mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -3,10 +3,22 @@ AUI.add(
 	function(A) {
 		var EVENT_CLICK = 'click';
 
-		var TPL_TASK = '<li>' +
-				'{task}' +
-				'<i class="icon-remove"></i>' +
-			'</li>';
+		var TPL_FINISHED = '<li>{taskFinished} ' +
+								'<button>' +
+									'<i class="icon-trash"></i>' +
+								'</button>' +
+						'</li>';
+
+		var TPL_HIST = '<tr>' +
```

Comment:
Each var name should hold a meaning. When I look at TPL_HIST and TPL_FINISHED, I have no idea what those templates hold.